variables.pkr.hcl

```diff
@@ -10,7 +10,7 @@ packer {
 locals {
   packages_al1    = "amazon-efs-utils ec2-net-utils acpid irqbalance numactl rng-tools docker-storage-setup"
   packages_al2    = "amazon-efs-utils ec2-net-utils acpid amazon-ssm-agent yum-plugin-upgrade-helper"
-  packages_al2022 = "amazon-efs-utils"
+  packages_al2022 = "amazon-efs-utils amazon-ec2-net-utils acpid"
```

Comment:
did we ever determine what we need `acpid` for?

Reply:
According to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/compute-optimized-instances.html

**Compute optimized instances**
> Instances built on the Nitro System should have acpid installed to support clean shutdown through API requests.
  
So we should install the package to support this instance type.